.travis.yml

```diff
@@ -10,6 +10,24 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 3.8.0" LEGACY=yes
   - rvm: 2.1.9
     env: PUPPET_GEM_VERSION="~> 4.9" DEPLOY_CANDIDATE=yes
+  - sudo: required
+    services: docker
+    rvm: '2.3.3'
+    env: PUPPET_INSTALL_VERSION="1.5.2" PUPPET_INSTALL_TYPE=agent BEAKER_set="centos-6-x86_64-docker"
```

Comment:
I think because at the time I wrote the original version of these specs, Puppet 4.5 was what was in the LTS release of Puppet Enterprise, I can update it now 😄